frontend/src/App.tsx

```diff
@@ -1,31 +1,26 @@
 import { useState } from 'react';
-import reactLogo from './assets/react.svg';
-import viteLogo from '../../../../../vite.svg';
 import './App.css';
+import MockPage from './components/mockPage/MockPage';
+import LoginPage from './components/loginPage/LoginPage';
+
+
+interface IUser {
+	name: string;
+	loggedIn: boolean;
+}
 
 export function App() {
-  const [count, setCount] = useState(0);
 
-  return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank" rel="noreferrer">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank" rel="noreferrer">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Sportify Me!</h1>
-      <div className="card">
-        <button type="button" onClick={() => setCount((number) => number + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">Click on the Vite and React logos to learn more</p>
-    </>
-  );
+	const noNameUser: IUser = {
+		name: "noName",
+		loggedIn: false
+	}
+	const [user, setUser] = useState<IUser>(noNameUser);
+
+	return (
+		<>
+			{!user.loggedIn && <LoginPage />}
```

Comment:
It would be better to return given component based on the logged condition. However, I propose to use dom-router for this functionality.